index.js

```diff
@@ -0,0 +1,60 @@
+const { Bot } = require('grammy');
+
+const bot = new Bot('5532232117:AAG_8EgKVkKd04srN7B5aHLKatC3biacIHc');
+bot.use((ctx, next) => {
+    ctx.state = {};
+    return next();
+})
+
+const middlewareMatchToSplit = function (ctx, next){
+    if(ctx.match === ''){
```

Comment:
You could check for falsy boolean here instead of empty string

Reply:
Done